src/librustdoc/html/static/js/src-script.js

```diff
@@ -71,16 +71,26 @@ function createDirEntry(elem, parent, fullPath, hasFoundFile) {
     return hasFoundFile;
 }
 
+window.rustdocCloseSourceSidebar = () => {
+    const toggleLabel = document.querySelector("#src-sidebar-toggle button");
```

Comment:
Nitpicking here but since we already have `toggleLabel` in some parent functions, maybe it could be passed as argument to this function and then:

```suggestion
    toggleLabel = toggleLabel || document.querySelector("#src-sidebar-toggle button");
```

Not sure what you think about it?